care/facility/api/viewsets/bed.py

```diff
@@ -65,7 +65,7 @@ def create(self, request, *args, **kwargs):
         validated_data = serializer.validated_data
         number_of_beds = validated_data.pop("number_of_beds", 1)
         # Bulk creating n number of beds
-        if number_of_beds > 1:
+        if number_of_beds > 1 and number_of_beds <= 100:
```

Comment:
@siddnikh it would be nice to raise a validation error for number_of_beds > 100

Reply:
Yeah, it's in the serializer file.